docs/content/preview/secure/authentication/host-based-authentication.md

```diff
@@ -25,7 +25,7 @@ showAsideToc: true
 
 YugabyteDB host-based authentication for YSQL manages access control for localhost, remote hosts, and clients. Using host-based authentication, you can define rules for access to localhost and remote clients based on IP addresses, authentication methods, and use of TLS (aka SSL) certificates.
 
-The default YugabyteDB `listen_addresses` setting only accepts connections from `localhost`. To allow remote connections, you must add client authentication records to the YB-TServer [`--ysql_hba_conf_csv`](../../../reference/configuration/yb-tserver/#ysql-hba-conf-csv) configuration flag. The flag works similarly to the `pg_hba.conf` file in PostgreSQL. The values include records that specify allowed connection types, users, client IP addresses, and authentication methods. These records are stored in the YugabyteDB `ysql_hba.conf` file.
+The default YugabyteDB `listen_addresses` setting only accepts connections from `localhost`. To allow remote connections, you must add client authentication records to the YB-TServer [--ysql_hba_conf_csv](../../../reference/configuration/yb-tserver/#ysql-hba-conf-csv) configuration flag. The flag works similarly to the `pg_hba.conf` file in PostgreSQL. The values include records that specify allowed connection types, users, client IP addresses, and authentication methods. These records are stored in the autogenerated YugabyteDB [ysql_hba.conf](#ysql-hba-conf-file) file.
```

Comment:
title: Host-Based Authentication
headerTitle: Host-Based Authentication
linkTitle: Host-Based Authentication

Change to :
title: Host-based authentication
headerTitle: Host-based authentication
linkTitle: Host-based authentication